ExploreASL_Master.m

```diff
@@ -17,7 +17,7 @@
 %                - ASL = Run the ASL Module (BOOLEAN, DEFAULT = 0)
 %                - POPULATION = Run the Population Module (BOOLEAN, DEFAULT = 0)
 %
-%   SkipPause    - TRUE = Pause workflow before ExploreASL pipeline (OPTIONAL, DEFAULT = FALSE)
+%   SkipPause    - FALSE = Pause workflow before ExploreASL pipeline (OPTIONAL, DEFAULT = FALSE)

```

Comment:
Haha, this is my fault. I guess the name of a Boolean should always explain a positive thing.
So this Boolean should be called `bPause`. I called this Boolean `bSkipPause` as you by default want to pause (from the perspective that the user starts ExploreASL).

Let's rename this parameter to `bPause`, that is defaulted to FALSE (no pause). 
But in some situations we might want to default to bPause = TRUE, to give the user the chance to look at the screen settings before continuing. Or shall we just remove this parameter at all, and instruct the "advanced user" (the one that won't use Maurice's GUI) to start running ExploreASL using the "load data only" option (as a sort of dummy start of ExploreASL)?

Reply:
I'm fine with both. But using false on default already sounds like a really good idea ;) Should be done in 3c2529f.